### ISSUE TITLE
Individual log-likelihoods

### ISSUE THREAD

**Issue by pmelchior** [pmelchior/spender#1]
The current code computes the loss (= negative log likelihood) summed over the entire batch:
https://github.com/pmelchior/spectrum-encoder/blob/8c021d4f339d75738a53d0ed29d4c19649946c97/model.py#L271

For outlier detection on the basis of the likelihood we'd like this to be computed for individual spectra, which means that the summation should only go over the spectral elements.

This can be done by 
```
torch.sum(0.5 * w * (x - spectrum_observed).pow(2), dim=1) + lognorm
```

The best way is to create an option for `loss`, e.g. `individual=False`, which gets passed on to the actual loss function. I'll implement this.

With that in place, you can check the current data set with the current model for outliers that have the highest loss.
